sale_coupon_multi_use/models/sale_coupon.py

```diff
@@ -0,0 +1,154 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleCoupon(models.Model):
+    """Extend to implement multi use coupon rule."""
+
+    _inherit = "sale.coupon"
+
+    # Takes value from related program (coupon_multi_use field), when
+    # it is generated.
+    multi_use = fields.Boolean("Multi Use", readonly=True)
```

Comment:
```suggestion
    multi_use = fields.Boolean(readonly=True)
```
fields string would be composed automatically like that

Reply:
I know, just like to be explicit here:)